Performance.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace LanguageServer\Tests;
```

Comment:
I'm hesitant to write an assert that a test runs in less than N seconds. You can really only compare performance between two runs on a given machine under controlled conditions.

If you want to do this in Travis or something (which would be awesome) maybe we could have it check out the previous version and run the test, then immediately check out the new version and run the test, and run several times and average the runs. But you still have to think about other things happening on the same machine, filesystem caching, other environmental things out of our control, so it might still end up flaky. I think doing this right is out of scope of this PR.